pylab/website/views.py

```diff
@@ -71,7 +71,8 @@ def about(request):
 
 def event_details(request, year, month, day, slug):
     event = get_object_or_404(Event, starts__year=year, starts__month=month, starts__day=day, slug=slug)
-    return render(request, 'website/event_details.html', {'event': event})
+    attendance = Attendance.objects.all().filter(event=event)
```

Comment:
You could pass model name instead of model to ```ForeignKey```, e.g. ```event = models.ForeignKey('Event')```. It works with models cross-referencing.